### ISSUE TITLE
Pretty icon in the status bar 

### ISSUE THREAD

**Issue by maukur** [TermiT/Flycut#285]
Could you please change the app icon? 

The new icon looks pretty nice:
<img width="345" alt="image" src="https://github.com/TermiT/Flycut/assets/8806716/0f5ac5ca-31cc-4a1c-966c-8ffb75c8fdae">

Icons: 
<img width="32" alt="com generalarcade flycut 16" src="https://github.com/TermiT/Flycut/assets/8806716/65c8d606-c0ff-4c0c-9252-fd231499ca04"> <img width="32" alt="com generalarcade flycut 16" src="https://github.com/TermiT/Flycut/assets/8806716/cb60e0a2-c629-44ea-a86a-d1c7db6009d0">

**Comment by Mauzzz0** [comment]
or add a "Show Icon in Menu Bar" checkbox option allowing users to un-check and disable the icon :)

**Comment by s-ja** [comment]
I was thinking the same thing.
I'd like to try some designs,
it would be great if there was a guide or something.

**Comment by thenktor** [comment]
Yes, please provide new icons. The current ones look like foreign objects in the menu bar.
